examples/src/main.ts

```diff
@@ -137,6 +142,10 @@ muya.on('json-change', (_changes) => {
 //   console.log(JSON.stringify([anchor.offset, focus.offset, path]))
 // })
 
+muya.domNode.addEventListener('input', () => {
```

Comment:
Why add this listener?

Reply:
Sorry, I made a wrong push after submitting the pull request.